src/db/sqlite/lrsql/sqlite/sql/ddl.sql

```diff
@@ -426,3 +426,33 @@ ALTER TABLE activity_profile_document DROP COLUMN last_modified;
 -- :command :execute
 -- :doc Convert `activity_profile_document.last_modified` to timestamp - 04
 ALTER TABLE activity_profile_document RENAME COLUMN last_modified_tmp TO last_modified;
+
+
+/* Migration 2023-07-21-00 - Add Reaction Table */
+
+-- :name create-reaction-table!
+-- :command :execute
+-- :doc Create the `reaction` table if it does not yet exist.
+CREATE TABLE IF NOT EXISTS reaction (
+  id         TEXT NOT NULL PRIMARY KEY, -- uuid
+  ruleset    BLOB NOT NULL,             -- serialized reaction spec
```

Comment:
BLOB because transit or something?

Reply:
Also I presume this is where templates will be added or are we doing them in another table with a FK?

Reply:
BLOB because that's what the other JSON stuff used. I think the templates should just go in as part of the same structure.

Reply:
ok sounds good. I forgot we didnt use a json type, since it doesnt buy us anything at all